docs/tutorials/remote_client.md

```diff
@@ -1,88 +1,111 @@
 # Podman remote-client tutorial
 
-## What is the remote-client
+## Introduction
+The purpose of the Podman remote-client is to allow users to interact with a Podman "backend" while on a separate client.  The command line interface of the remote client is exactly the same as the regular Podman commands with the exception of some flags being removed as they do not apply to the remote-client.
 
-First and foremost, the remote-client is under heavy development.  We are adding new
-commands and functions frequently.  We also are working on a rootless implementation that
-does not require privileged users.
+The remote client uses a client-server model, taking advantage You need Podman installed on a Linux machine or VM that also has the SSH daemon running. On the local operating system, when you execute a Podman command, Podman connects to the server via SSH. It then connects to the Podman service by using systemd socket activation, and hitting our [Rest API](https://docs.podman.io/en/latest/_static/api.html). The Podman commands are executed on the server. From the client's point of view, it seems like Podman runs locally.
 
-The purpose of the Podman remote-client is to allow users to interact with a Podman "backend"
-while on a separate client.  The command line interface of the remote client is exactly the
-same as the regular Podman commands with the exception of some flags being removed as they
-do not apply to the remote-client.
+This tutorial is for running podman remotely on Linux. If you are using a Mac or a Windows PC, please follow the [Mac and Windows tutorial](https://github.com/containers/podman/blob/master/docs/tutorials/mac_win_client.md)
```

Comment:
```suggestion
This tutorial is for running Podman remotely on Linux. If you are using a Mac or a Windows PC, please follow the [Mac and Windows tutorial](https://github.com/containers/podman/blob/master/docs/tutorials/mac_win_client.md)
```

Reply:
and anywhere else that I might miss where "Podman" is used except in actual commands.